electron/preload/index.ts

```diff
@@ -289,6 +294,9 @@ contextBridge.exposeInMainWorld("electronStore", {
   updateChatHistory: (chatHistory: ChatHistory) => {
     ipcRenderer.invoke("update-chat-history", chatHistory);
   },
+  updateAllChatHistories: (chatID: string) => {
```

Comment:
Given that this calls the remove handler, perhaps it should be called removeChatHistory?